org.eclipse.jdt.ls.tests/pom.xml

```diff
@@ -56,6 +56,11 @@
 										<version>0.0.0</version>
 										<type>eclipse-feature</type>
 									</dependency>
+									<dependency>
+										<artifactId>org.eclipse.jdt.java16patch</artifactId>
```

Comment:
why do we need a patch still? Isn't it included in 2021-06-M1 already?

Reply:
D'oh, shouldn't be needed. Let me see if I can just get rid of it.